Synview-frontend/src/slices/inviteMentorModalSlice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from "@reduxjs/toolkit";
+import type { PayloadAction } from "@reduxjs/toolkit";
+
+interface inviteMentorState {
```

Comment:
nit: same here, should be `InviteMentorState` since this is not an instance variable or a const variable (don't ask me why people call them "const variable"s but they do :) )